python/ray/tests/test_basic_3.py

```diff
@@ -203,13 +200,12 @@ def f():
 
     # This will never finish without fair queueing of {f, g, h}:
     # https://github.com/ray-project/ray/issues/3644
-    timeout = 510.0 if sys.platform == "win32" else 60.0
+    timeout = 60.0
```

Comment:
Updated because, it never times out under 60.0s. If it does anywhere else on Windows then it might be innate to the OS and I think everything is fine with ray's code. It takes only one path which consumes appropriate amount of time i.e., 20.0s to 45s which is fine and well under the time limits.